tests/cases/conformance/types/thisType/thisTypeInFunctions2.ts

```diff
@@ -0,0 +1,26 @@
+interface Arguments {
```

Comment:
also a test for the contextual type of the object literal, without the `this:any`